pkg/controller/jobs/pod/pod_controller_test.go

```diff
@@ -2098,14 +2105,12 @@ func TestReconciler(t *testing.T) {
 			reconciler := NewReconciler(kClient, recorder)
 
 			for i := range tc.pods {
```

Comment:
Reconcile should only be called once now, instead of in a loop

Reply:
Updated the test in 8568b66b